testing/simulator/src/cli.rs

```diff
@@ -20,15 +20,15 @@ pub fn cli_app() -> Command {
                         .short('n')
                         .long("nodes")
                         .action(ArgAction::Set)
-                        .default_value("3")
+                        .default_value("2")
                         .help("Number of beacon nodes"),
                 )
                 .arg(
                     Arg::new("proposer-nodes")
                         .short('p')
                         .long("proposer-nodes")
                         .action(ArgAction::Set)
-                        .default_value("3")
+                        .default_value("2")
```

Comment:
This should be `1` if we want to have 4 nodes total (2 beacon nodes + 1 proposer node + 1 boot node)

```suggestion
                        .default_value("1")
```